apcd-cms/src/apps/admin_regis_table/templates/list_registrations.html

```diff
@@ -77,6 +80,14 @@ <h1>List Registrations</h1>
       }
     })
   }
+  function openAction(reg_id) {
```

Comment:
This is really slick! But when the modals didn't open on the second and third row of options, it was a little hard for me to debug. I know I'd appreciate it if there was a comment explaining what's going on, or the variable names were a little more unique. It's hard for my brain to figure out the difference between dropdown, selectedOption, and actionsDropdown. Good work, you're getting a lot done here with just a couple lines.